src/freenet/clients/fcp/SimpleProgressMessage.java

```diff
@@ -70,6 +72,11 @@ public double getFetchedBlocks(){
 		return event.succeedBlocks;
 	}
 	
+	public Date getLatestSuccess() {
+        // clone() because Date is mutable
+	    return event.latestSuccess != null ? (Date)event.latestSuccess.clone() : null;
```

Comment:
Whitespace fixed by 3418c7daa580672fa0239b41113ac27ec388f2d2.
Done this since I assume that people prefer having 1468 soon to having an answer about whether billing probably less than 5 EUR here is OK.